exporter/fileexporter/file_exporter.go

```diff
@@ -112,15 +116,57 @@ func exportMessageAsBuffer(e *fileExporter, buf []byte) error {
 	if err := binary.Write(e.file, binary.BigEndian, data); err != nil {
 		return err
 	}
+
 	return nil
 }
 
+func startFlusher(e *fileExporter) {
```

Comment:
Rather than pass the exporter into the function, let's make this a receiver method.
```suggestion
func (e *fileExporter) startFlusher() {
```

Reply:
Understood